x-pack/plugins/canvas/server/lib/get_request.js

```diff
@@ -3,15 +3,24 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
+import boom from 'boom';
+import { API_ROUTE } from '../../common/lib/constants';
 
 export function getRequest(server, { headers }) {
-  const basePath = server.config().get('server.basePath') || '/';
+  const url = `${API_ROUTE}/ping`;
 
   return server
     .inject({
-      method: 'GET',
-      url: basePath,
+      method: 'POST',
+      url,
       headers,
     })
-    .then(res => res.request);
+    .then(res => {
+      if (res.statusCode !== 200) {
+        console.error(new Error(`Auth request failed: [${res.statusCode}] ${res.result.message}`));
```

Comment:
i18n?  And do we usually publish to the console?  Just curious if that is a standard practice at Elastic; it hasn't been in my roles elsewhere.

Reply:
I don't know if we have a standard or not. But far too often things fail with no output at all, so I tend to err on the side of too much info. This request doesn't go over the wire, so there's no other indication of *why* auth failed, which is really frustrating in dev mode. I'll add a conditional so this doesn't show up in production mode though.